src/index.html

```diff
@@ -1,17 +1,155 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport" content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>HTML Form</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post"
+    name="mainForm">
+      <fieldset class="form_block" name="Personal">
+        <legend>Personal information</legend>
+        <div class="form_line">
+          <label id="Surname">
+            Surname:
+            <input type="text" name="Surname"
+             required="required"
+            >
+          </label>
+        </div>
+        <div class="form_line">
+          <label>Name:
+            <input type="text" name="Name"
+            required="required"
+            >
```

Comment:
```suggestion
            <input
              type="text"
              name="Surname"
              required="required"
            >
          </label>
        </div>
        <div class="form_line">
          <label>Name:
            <input
              type="text"
              name="Name"
              required="required"
            >
```
[checklist p.3](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md)

fix code style all over the page before requesting review